README.md

```diff
@@ -133,7 +133,7 @@ If you like this project please consider starring it.
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-We would like to thank [Microverse](www.microverse.com).
+I would like to thank [Microverse](www.microverse.com).
```

Comment:
- [ ] Kindly update the link in the acknowledgment section, this is currently broken, all links in your readme should not be broken